internal/cmd/egctl/testdata/translate/out/default-resources.all.yaml

```diff
@@ -1082,6 +1082,13 @@ xds:
             routes:
             - match:
                 prefix: /
+              metadata:
+                filterMetadata:
+                  io.envoyproxy.gateway.metadata:
```

Comment:
`metadata.filterMetadata.io.envoyproxy.gateway.metadata` seems a bit redundant, maybe just use`io.envoyproxy.gateway` as the key?